ccd-definitions-jsonnet/CaseEvent.jsonnet

```diff
@@ -0,0 +1,24 @@
+local base = import 'lib/base.libsonnet';
+local constants = import 'lib/constants.libsonnet';
+
+local DefaultCaseEvent = base.LiveDates + {
```

Comment:
JSON templates are an alternative to functions used in `CaseEventToFields.jsonnet` file.